runtime-sdk/src/context.rs

```diff
@@ -76,6 +76,9 @@ pub trait Context {
         self.mode() == Mode::SimulateTx
     }
 
+    /// Runtime ID.
+    fn runtime_id(&self) -> &Namespace;
```

Comment:
I can add this, but I'd still like to keep the `runtime_id()` method, since it will probably see much more use than the rest of the host info.

Reply:
Sure, that's fine, it should just be a default implementation on the trait that uses `self.host_info()` for doing the actual work. And so the contexts only need to store a reference to `host_info` instead of a separate one to runtime ID.